postgrenosql/README.md

```diff
@@ -0,0 +1,24 @@
+<!--
+Copyright (c) 2012 - 2016 YCSB contributors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you
+may not use this file except in compliance with the License. You
+may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+implied. See the License for the specific language governing
+permissions and limitations under the License. See accompanying
+LICENSE file.
+-->
+
+## Quick Start
+
+This section describes how to run YCSB on PostgreSQL as document store.
+ 
+## Download the newest Postgres Server Version
+
+ToDo
```

Comment:
Could you add a quick example of running things. It's fine to skip "how do I install postgres" and just state a required version you have to be running. If there's some server side configuration that has to be altered to get the nosql feature, then you ought to call that out.